commit fd94cd22f1d03872bf84ace2f9c676fc82bdbcf6
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-08-15T11:26:46-07:00 (8 weeks ago)
    
    TOOLS-2307 Update the copyright example in the eng docs
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/docs/index.md b/docs/index.md
index a11ec38..1e21a67 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -10,7 +10,7 @@ apisections:
 -->
 
 <!--
-    Copyright (c) 2019, Joyent, Inc.
+    Copyright 2019 Joyent, Inc.
 -->
 
 # Joyent Engineering Guide
@@ -318,7 +318,7 @@ When modifying existing code, the year should be updated to be the
 current year that the file was modified. There should only be a single
 year, not a list. For example:
 
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright 2019 Joyent, Inc.
 
 
 ## Testing
